src/Auth0Client.ts

```diff
@@ -135,6 +143,7 @@ const getCustomInitialOptions = (
     redirect_uri,
     scope,
     useRefreshTokens,
+    useCookiesForTransactions,
```

Comment:
This was inadvertently being included in calls to the token endpoint - adding it here removes it.